buildSrc/src/main/groovy/nullaway.java-test-conventions.gradle

```diff
@@ -118,3 +102,23 @@ def testJdk21 = tasks.register("testJdk21", Test) {
 tasks.named('check').configure {
     dependsOn testJdk21
 }
+
+
+// Share the coverage data to be aggregated for the whole product
```

Comment:
This change is required so that test coverage when running tests on JDK 21 gets reported to CodeCov.